SPECS-EXTENDED/python-cov-core/python-cov-core.spec

```diff
@@ -0,0 +1,170 @@
+%{?python_enable_dependency_generator}
+# Created by pyp2rpm-1.0.1
+%global pypi_name cov-core
+%global summary Plugin core for use by pytest-cov, nose-cov and nose2-cov
+%global _description \
+This is a lib package for use by pytest-cov, nose-cov and nose2-cov. \
+If you're developing a coverage plugin for a test framework then you \
+probably want one of those.
+
+Name:           python-%{pypi_name}
+Version:        1.15.0
+Release:        21%{?dist}
+Summary:        %{summary}
+
+License:        MIT
+URL:            http://bitbucket.org/memedough/cov-core/overview
+# url Source seems to be broken
```

Comment:
is it the case ? could you find the right URL ? it will be needed for cgmanifest.json